apps/src/applab/applab.js

```diff
@@ -1213,6 +1219,11 @@ Applab.encodedFeedbackImage = '';
  * @param {ApplabInterfaceMode} mode
  */
 function onInterfaceModeChange(mode) {
+  Applab.setWorkspaceMode(mode);
+
+  var showDivApplab = (mode !== ApplabInterfaceMode.DESIGN);
+  Applab.toggleDivApplab(showDivApplab);
```

Comment:
I had looked into this and it looks like there are 3 props we'd want to move inside of DesignWorkspace.jsx before we can remove it from its ProtectedStatefulDiv:
* `element: element || null`,
    * --> add elementId as new redux state (or maybe just React state to start)
* `elementIdList: Applab.getIdDropdownForCurrentScreen(),`
  * --> replace with `currentScreenId` from existing redux state
* `isDimmed: Applab.running`
  * --> replace with `runState.isRunning` from existing redux state